src/client/app/components/HeaderComponent.jsx

```diff
@@ -22,16 +22,21 @@ export default function HeaderComponent(props) {
 	};
 	const loginLinkStyle = {
 		// Displays the login button link only if the user is not logged in or is explicitly told to display by the parent component
-		display: localStorage.getItem('token') || props.renderLoginButton === 'false' ? 'none' : 'inline'
+		display: localStorage.getItem('token') || props.renderLoginButton === false ? 'none' : 'inline'
```

Comment:
Maybe wrap the `||` with some parentheses to make it really obvious, but I begrudgingly agree this is probably ok.